examples/ilu-preconditioned-solver-iterative-trisolves/ilu-preconditioned-solver-iterative-trisolves.cpp

```diff
@@ -55,20 +55,25 @@ int main(int argc, char *argv[])
     // Print version information
     std::cout << gko::version_info::get() << std::endl;
 
-    // Figure out where to run the code
+    // Figure out where to run the code and how many block-Jacobi sweeps to use
     std::shared_ptr<gko::Executor> exec;
-    if (argc == 1 || std::string(argv[1]) == "reference") {
+    unsigned int sweeps;
+    if (argc == 2 || std::string(argv[2]) == "reference") {
```

Comment:
Can running `./ilu-preconditioned-solver-iterative-trisolves` get the `Usage:` message?
does argv[2] not touch unused memory?